src/arduino/affichage.cpp

```diff
@@ -0,0 +1,128 @@
+/*=============================================================================
+ * @file affichage.cpp
+ *
+ * COVID Respirator
+ *
+ * @section copyright Copyright
+ *
+ * Makers For Life
+ *
+ * @section descr File description
+ *
+ * Fichier implémentant les fonctionnalitées liés à l'affichage
+ */
+
+// INCLUDES ===================================================================
+
+// Associated header
+#include "affichage.h"
+
+// External libraries
+#include <LiquidCrystal.h>
+
+// Internal libraries
+#include "common.h"
+
+// FUNCTIONS ==================================================================
+
+void startScreen()
+{
+  switch (screenSize)
+  {
+  case ScreenSize::CHARS_16:
+  {
+    screen.begin(16, 2);
+    break;
+  }
+  case ScreenSize::CHARS_20:
+  {
+    screen.begin(20, 2);
+    break;
+  }
+  default:
+  {
+     screen.begin(16, 2);
```

Comment:
There is a code duplication here. Why not remove the `ScreenSize::CHARS_16` case as it is supposed to be identical to `default`?

_The same duplication pattern happens 3 times in this file_

Reply:
Done